apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-welcome-tour/src/tour-card.js

```diff
@@ -7,84 +7,140 @@ import './public-path';
 import './style-tour.scss';
 import PaginationControl from './pagination';
 import minimize from './icons/minimize';
+import thumbsUp from './icons/thumbs_up';
+import thumbsDown from './icons/thumbs_down';
 
 /**
  * External dependencies
  */
+import classNames from 'classnames';
 import { Button, Card, CardBody, CardFooter, CardMedia, Flex } from '@wordpress/components';
 import { close } from '@wordpress/icons';
+import { useEffect } from '@wordpress/element';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+
+// eslint-disable-next-line react-hooks/exhaustive-deps
+const useEffectOnlyOnce = ( func ) => useEffect( func, [] );
```

Comment:
This is annoying, but I just tried it myself and my eslint set up is also complaining about using `[]` in the dependency list. I thought eslint wasn't supposed to complain when the array was empty because this is a well known pattern 😡

Reply:
No action item here, just me venting 😓

Reply:
My thoughts exactly.   It would be cool if there was a `useEffectOnlyOnce` in core, as I would think this is a sometimes necessary!

Reply:
Maybe `useOnMount()`? Since we are looking for that old `onMount()` behaviour in this case.